packages/concerto-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@accordproject/concerto-core",
-  "version": "0.82.7",
+  "version": "1.0.0",
```

Comment:
This should not be done by hand (this will prevent the use of the publish scripts). Please revert.

Reply:
We will have to remember to version in the change log by hand.